library/src/main/java/com/novoda/downloadmanager/lib/DownloadInfo.java

```diff
@@ -479,12 +483,12 @@ public boolean isOnCache() {
                 || mDestination == Downloads.Impl.DESTINATION_CACHE_PARTITION_PURGEABLE);
     }
 
-    public Uri getMyDownloadsUri() {
-        return ContentUris.withAppendedId(Downloads.Impl.CONTENT_URI, mId);
+    public Uri getMyDownloadsUri(String authority) {
```

Comment:
this is publicly accessible, we don't want the client to have to pass their authority. Either we make this method package? or we do somethign else